quinn-proto/src/coding.rs

```diff
@@ -92,6 +99,7 @@ impl Codec for Ipv6Addr {
     }
 }
 
+#[doc(hidden)]
```

Comment:
Could we make this and below `pub(crate)` instead?

Reply:
done, seems to work out